### ISSUE TITLE
Implement column filter for grid

### ISSUE THREAD

**Issue by romaninsh** [atk4/ui#434]
Grid has QuickSearch, but it does not allow any precise filtering on columns.

The idea is to implement a mechanism, you could click on the column menu (implemented by #432) 

<img width="673" alt="screen shot 2018-04-12 at 18 38 35" src="https://user-images.githubusercontent.com/453929/38694929-cd0af5d2-3e82-11e8-9888-f4f3546f87c8.png">

**Comment by DarkSide666** [comment]
related #420

**Comment by venditiogroup** [comment]
We use the filter column feature extensively. It's works quite  well, but the Airtable implementation suggested on #420 is more powerful (allows OR) and is easier to quickly see / modify criteria. Is there a way I can sponsor this feature?

Thanks

**Comment by romaninsh** [comment]
@venditiogroup yes, if you could contact me at support@agiletoolkit.org, (or "romaninsh" on skype) we can discuss your requirements.

**Comment by romaninsh** [comment]
This was implemented in https://github.com/atk4/ui/pull/448
